at_events_flutter/lib/models/enums_model.dart

```diff
@@ -24,7 +27,7 @@ DateTime? startTimeEnumToTimeOfDay(String startTimeEnum, DateTime? startTime) {
   }
 }
 
-// ignore: missing_return
+// ignore: missing_return, body_might_complete_normally_nullable
 DateTime? endTimeEnumToTimeOfDay(String endTimeEnum, DateTime? endTime) {
```

Comment:
Can we return null at the end of `endTimeEnumToTimeOfDay `  function and remove remove ignore comment.